src/main/index.ts

```diff
@@ -121,6 +121,8 @@ app.on("second-instance", (event, argv) => {
 });
 
 app.on("ready", async () => {
+  await injectSystemCAs();
```

Comment:
Should we call this earlier to match previous behaviour more closely?

Reply:
Where do you suggest? Or we should use the old "require at top" of renderer.ts/main.ts instead of export a function?

Reply:
I mean before `ready` event (after imports).

Reply:
OK. thanks

Reply:
Moved at the top (as before this PR) https://github.com/lensapp/lens/pull/4185/commits/aa7645347fe5cf25d2456f9f6a6a67e6cefd71c3